docs/docs/hardware/input-devices/I2CRJ45.md

```diff
@@ -43,4 +53,5 @@ TODO - Jon to supply some FAQ's
 
 ## Compatible With
 - Rack32 [Link](/docs/hardware/controllers/rack32.md) **<-- Best Choice**

```

Comment:
Remove "Best Choice" - the regulator problems with the Rack32 mean it is no longer the best choice ;)